package-lock.json

```diff
@@ -1,8 +1,4433 @@
 {
   "name": "git-activity",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
You need to discard all the  `package-lock.json` changes, these cannot be in this PR as there are no new dependencies. Please remove those by resetting your `package-lock.json` to the version which is on the `master branch`.